### ISSUE TITLE
Renamespaced content overwrites . seperators with / & doesnt renamespace files with a £

### ISSUE THREAD

**Issue by charliehw** [BladeRunnerJS/brjs#1315]
I'm having two issues when importing an app using 0.15.3. In one case, a test js file in the app is not having namespaced references replaced to the app's namespace so the tests fail. Other test files that exist in the same folder are having their references namespaced. I can't see how the first file is different to the others.

The second issue occurs when namespacing some xml that exists in a test file as a string:
    `"<caplinx.fxexecution.tilelayout.tile-set></caplinx.fxexecution.tilelayout.tile-set>"`
becomes
    `"<novo.fxexecution.tilelayout.tile-set></novo/fxexecution.tilelayout.tile-set>"`

The double / in the output causes a parse error and the tests fail.

I have sent the app I'm struggling with to @ioanalianabalas

**Comment by ioanalianabalas** [comment]
1. It seems that the test js file is getting correctly renamespaced in our `develop` branch but not in `master` (corresponding to the latest release, 0.15.3). So this will most likely be fixed in the next release.

2. This issue is present on both the `develop` and `master` branches.

**Comment by charliehw** [comment]
I have tracked down the first issue to the inclusion of "£" in a string in the test file. If I remove this single character and import the app again, the namespace is converted properly.

**Comment by ioanalianabalas** [comment]
Fixed by #1317.

**Comment by thanhc** [comment]
@andyberry88 we've verified that the file is encoded UTF8 so its not an encoding issue, could it be similar to what I had to do with the spectest issue where we aren't reading in the file with the right encoding see commit c3b2fc82d89abd

**Comment by andyberry88** [comment]
I've update the title to reflect the current known issue. @charliehw can you confirm that it was a `$` somewhere in the file that was causing it to not be namespaced?

**Comment by ccpetercc** [comment]
I've tested the `renamespace-with-separators-1315` branch running gradlew brjs-sdk:distZip and using that as a dependency in CT (branch upgrdage-brjs-1.0). I then imported Charlie's fxtrader App which causes the bugs (using Import Motif from ZIP) setting the namespace to fxtrader2.

The issue in /ct-core/apps/fxtrader2/fxexecution-bladeset/blades/blocktrade/tests/test-unit/js-test-driver/tests/upload/presentation/ExcelImportControlTest.js regarding the £ character does not appear to have been fixed - 

After importing
```
line 7 is:
fields: caplinx.fxexecution.blocktrade.manager.presentation.Manager.prototype._generateExcelFields()
when it should be:
fields: fxtrader2.fxexecution.blocktrade.manager.presentation.Manager.prototype._generateExcelFields()
```
Having set the namespace to fxtrader2


The issue in /ct-core/apps/fxtrader2/fxexecution-bladeset/blades/tilelayout/tests/test-unit/js-test-driver/tests/fxtrader2/fxexecution/tilelayout/TileSetGroupTest.js

does appear to have been fixed. The XML is now correct.

I wasn't able to test this extensively. Errors are thrown by Charlies fxtrader app and it is not possible to run successfully. This is most likely because I imported the App into the CT branch, upgrade-brjs-1.0, which is required to test the BRJS SDK dependency.
